frontend/src/redux/userArtworkActions.js

```diff
@@ -0,0 +1,142 @@
+import PlaceholderImage from '../components/dashboard/images/colorful.jpeg';
+
+function fakeGetUserArtworks() {
+  return new Promise((resolve) => {
+    // Resolve after a timeout so we can see the loading indicator
+    setTimeout(
+        () =>
+          resolve({
+            artworks: [
```

Comment:
DIY principal: we could extract a constant here. 

But it these artworks are just temporary fake data, forget about the comment above.